addons/mail/static/tests/web/follower_list_menu_tests.js

```diff
@@ -289,6 +290,33 @@ QUnit.test("Load 100 recipients at once", async (assert) => {
     assert.containsNone($, ".o-mail-RecipientList span:contains(Load more)");
 });
 
+QUnit.test("Load recipient without email", async (assert) => {
+    const pyEnv = await startServer();
+    const [partnerId_1, partnerId_2] = pyEnv["res.partner"].create([
+        { name: "Luigi" },
+        { name: "Mario" },
+    ]);
+    pyEnv["mail.followers"].create([
+        {
+            is_active: true,
+            partner_id: pyEnv.currentPartnerId,
+            res_id: partnerId_1,
+            res_model: "res.partner",
+        },
+        {
+            is_active: true,
+            partner_id: partnerId_2,
+            res_id: partnerId_1,
+            res_model: "res.partner",
+        },
+    ]);
+    const { openFormView } = await start();
+    await openFormView("res.partner", partnerId_1);
+    await click("button:contains(Send message)");
```

Comment:
`:contains` should preferably be replaced by `text:` param.
I didn't even remember it was still supported.

Reply:
the problem is click in this file is imported from "@mail/../tests/helpers/test_utils" rather than the new click we have for now. Otherwise I will have to do something like 
`import { contains, click as WebClick } from "@web/../tests/utils";` not sure if it is preferred.

Reply:
2nd option, the new helper is better in every way